public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/AlertManagerRouting.tsx

```diff
@@ -0,0 +1,122 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Alert, CollapsableSection, Icon, Link, LoadingPlaceholder, Stack, Text, useStyles2 } from '@grafana/ui';
+import { AlertManagerDataSource } from 'app/features/alerting/unified/utils/datasource';
+import { createUrl } from 'app/features/alerting/unified/utils/url';
+
+import { useContactPointsWithStatus } from '../../../contact-points/useContactPoints';
+
+import { ContactPointDetails } from './contactPoint/ContactPointDetails';
+import { ContactPointSelector } from './contactPoint/ContactPointSelector';
+import { MuteTimingFields } from './route-settings/MuteTimingFields';
+import { RoutingSettings } from './route-settings/RouteSettings';
+
+interface AlertManagerManualRoutingProps {
+  alertManagerContactPoint: {
```

Comment:
I suggest to split this up a bit, there's three parts here that would translate well to props;

`alertmanager`, `selectedContactPoint` and `routingOptions` (which consists of grouping and timings).